blazar_dashboard/content/leases/templates/leases/_widget_capabilities.html

```diff
@@ -17,18 +17,19 @@
 function capabilitiesjs() {
     'use strict';
 
-    var capabilityNames = [];
-    var capabilityValues = {};
-    var capabilityRows = [];
+    var hostCapabilityNames = [];
+    var hostCapabilityValues = {};
 
     var criteriaCounter = 0; // gives distinct values to the input names
 
     // initial load of names
     var xhr = new XMLHttpRequest();
     xhr.onreadystatechange = function() {
+        console.log(this);
```

Comment:
Please remove all debug stuff like this, unless it's really useful for debugging in a general use case